docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Node Version Manager (nvm)
+description: Using Flox to replace nvm
+---
+
+# Node Version Manager (nvm) migration guide
+
+Flox is an environment and package manager that allows you install software from an extensive catalog into individual [environments][environment_concept]{:target="\_blank"}, each of which usually corresponds to a software project. Flox gives you the opportunity to simplify the development workflow used by your team, consolidating various functions into a single command: `flox activate`.
+
+Just like [nvm](https://github.com/nvm-sh/nvm){:target="\_blank"}, Flox allows you to install and activate different versions of Node.js, allowing developers to switch versions as project requirements dictate.
```

Comment:
What's with the `{:target="\_blank"}`?

Reply:
It is a good practice to open a new tab / window when linking offsite, I like this! Perhaps mkdocs can automate.

Reply:
Yeah, I would for sure open all external links in a new tab. My personal preference is also to open other Flox docs links in new tabs, but I can drop that if it's non-standard and potentially confusing to the user.